__tests__/app.test.js

```diff
@@ -200,3 +197,47 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=votes&order=asc", () => {
+  test("gives status of 200 and responds with all articles sorted by votes and ordered by ascending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes&order=asc")
+      .expect(200)
+      .then(({ body: { allArticles } }) => {
+        console.log(allArticles)
+        allArticles.forEach((article) => {
+          expect(Object.keys(article)).toHaveLength(8);
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+
+        expect(allArticles).toBeSortedBy("votes", { ascending: true });
+      });
+  });
+
```

Comment:
It would be a good idea to also test for the default sort order (if you haven't already on this endpoint)